.storybook/webpack.config.js

```diff
@@ -1,21 +1,54 @@
 const path = require('path');
+const webpack = require('webpack')
 
 const srcPath = path.resolve(__dirname, '..', 'src')
 
-module.exports = (baseConfig) => {
+module.exports = (baseConfig, env) => {
   baseConfig.module.rules.push({
-    test: /\.s?css$/,
+    test: /\.scss$/,
+    include: path.resolve('src'),
     use: [
-      'style-loader',
+      require.resolve('style-loader'),
       {
-        loader: 'css-loader',
+        loader: require.resolve('css-loader'),
         options: {
           importLoaders: 2,
-        }
+        },
+      },
+      {
+        // Options for PostCSS as we reference these options twice
+        // Adds vendor prefixing based on your specified browser support in
+        // package.json
+        loader: require.resolve('postcss-loader'),
+        options: {
+          // Necessary for external CSS imports to work
+          // https://github.com/facebook/create-react-app/issues/2677
+          ident: 'postcss',
+          plugins: () => [
+            require('postcss-flexbugs-fixes'),
+            require('autoprefixer')({
+              browsers: [
+                '>1%',
+                'last 4 versions',
+                'Firefox ESR',
+                'not ie < 11',
+              ],
+              flexbox: 'no-2009',
+            }),
+          ],
+        },
+      },
+      {
+        loader: require.resolve('sass-loader'),
+        options: {
+          includePaths: [
+            path.resolve('src'),
+          ],
+        },
       },
-      'sass-loader'
     ],
   })
```

Comment:
isn't it possible to reuse these configs for loaders in both webpack configs?

Reply:
It looks like reusing requires more effort (and reduces webpack.config maintainability) than just copy-pasting on demand.